transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -162,6 +167,9 @@ public final int sendToAddress(long memoryAddress, int pos, int limit, InetAddre
         if (res >= 0) {
             return res;
         }
+        if (res == ERROR_ECONNREFUSED_NEGATIVE) {
+            throw new PortUnreachableException("sendto failed");
```

Comment:
(In reference to your previous comment) Sendto is used in other places but it is not used here. Seems confusing to reference an unused native method.